samples/java-eventsourced-customer-registry/src/main/proto/customer/view/customer_view.proto

```diff
@@ -67,15 +67,18 @@ service CustomerByName {
     };
   }
 
+  // tag::ignore-other-events[]
   rpc IgnoreOtherEvents(google.protobuf.Any) returns (api.Customer) {
     option (kalix.method).eventing.in = {
-      event_sourced_entity: "customers" // <5>
+      event_sourced_entity: "customers"
+      ignore: true
      };
      option (kalix.method).view.update = {
        table: "customers"
-       transform_updates: true // <6>
+       transform_updates: true
      };
   };
+  // end::ignore-other-events[]
```

Comment:
It's kind of awkward that we need to add 4 annotations for a method that is here just to be ignored. 
Are the `view.update` annotations needed? 

A service level annotation saying that it's partial will be handy.

Reply:
Good point. It works fine without the update annotation for ignore methods. I have removed them in https://github.com/lightbend/kalix-jvm-sdk/pull/1039/commits/2bf36dc83a0d4df9f7175191f2dd94cd4103fc61